gensim/test/test_fasttext_wrapper.py

```diff
@@ -120,6 +122,37 @@ def testLoadFastTextFormat(self):
         self.assertEqual(self.test_model.wv.syn0_all.shape, (self.test_model.num_ngram_vectors, model_size))
         self.model_sanity(model)
 
+    def testLoadFastTextNewFormat(self):
+        #Test model successfully loaded from fastText (new format) .vec and .bin files
+        new_model = fasttext.FastText.load_fasttext_format(self.test_new_model_file)
+        vocab_size, model_size = 1763, 10
+        self.assertEqual(self.test_new_model.wv.syn0.shape, (vocab_size, model_size))
```

Comment:
`test_new_model` is defined here in [line 38](https://github.com/prakhar2b/gensim/blob/58a66c2b6c704d668f86bf7e4c4a59c86beffa4a/gensim/test/test_fasttext_wrapper.py#L38). Only new model is being checked in this function `testLoadFastTextNewFormat`. Please compare it with function `testLoadFastTextFormat` to see the context.